qmat/qcoeff/butcher.py

```diff
@@ -300,3 +324,44 @@ class SDIRK54(RK):
 
     @property
     def order(self): return 4
+
+@registerRK
+class Heun_Euler(RKEmbedded):
+    """
+    Second order explicit embedded Runge-Kutta method.
+    """
+    A = [[0, 0],
+         [1, 0]]
+    b = [[0.5, 0.5], [1, 0]]
+    c = [0, 1]
+
+    @property
+    def order(self): return 2
+
+    @property
+    def orderSecondary(self): return 1
+
+@registerRK
+class Cash_Karp(RKEmbedded):
+    """
+    Fifth order explicit embedded Runge-Kutta. See [here](https://doi.org/10.1145/79505.79507).
+    """
+    c = [0, 0.2, 0.3, 0.6, 1.0, 7.0 / 8.0]
+    b = [
+            [37.0 / 378.0, 0.0, 250.0 / 621.0, 125.0 / 594.0, 0.0, 512.0 / 1771.0],
+            [2825.0 / 27648.0, 0.0, 18575.0 / 48384.0, 13525.0 / 55296.0, 277.0 / 14336.0, 1.0 / 4.0],
+        ]
+    A = np.zeros((6, 6))
```

Comment:
Like this approach btw : maybe this could be suggested in the docs for adding RK scheme :wink: